portal-frontend/src/app/features/public/notice-of-intent/submission/parcel/parcel.component.html

```diff
@@ -58,7 +58,40 @@ <h4>Parcel #{{ parcelInd + 1 }}</h4>
         <app-no-data *ngIf="parcel.crownLandOwnerType === null"></app-no-data>
       </div>
     </ng-container>
-    <div class="full-width owner-information">
+    <ng-container *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+      <h5 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government Parcel Contact</h5>
+      <ng-container *ngIf="parcel.owners[0]">
+        <div class="subheading2 grid-1">First Name</div>
+        <div class="grid-double">
+          {{ parcel.owners[0].firstName }}
+          <app-no-data *ngIf="!parcel.owners[0].firstName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Last Name
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].lastName }}
+          <app-no-data *ngIf="!parcel.owners[0].lastName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Ministry or Department
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].organizationName }}
+          <app-no-data *ngIf="!parcel.owners[0].organizationName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Crown Type
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].crownLandOwnerType === 'provincial' ? 'Provincial Crown' : '' }}
+          {{ parcel.owners[0].crownLandOwnerType === 'federal' ? 'Federal Crown' : '' }}
+          <app-no-data *ngIf="!parcel.owners[0].crownLandOwnerType"></app-no-data>
+        </div>
+      </ng-container>
+      <app-no-data *ngIf="!parcel.owners[0]"></app-no-data>
+    </ng-container>
+    <div class="full-width owner-information" *ngIf="parcel.ownershipTypeCode !== PARCEL_OWNERSHIP_TYPES.CROWN">
```

Comment:
Same as application and noi

Reply:
Same again, public view has not been updated yet, code still has unnecessary *ngIf